lib/Zonemaster/Backend/Validator.pm

```diff
@@ -65,7 +65,7 @@ sub priority {
     return joi->integer;
 }
 sub profile_name {
-    return joi->string->regex("^(?![-_])[a-zA-Z0-9-_]{1,32}(?<![-_])\$")->min(1)->max(32);
+    return joi->string->regex('^[a-zA-Z0-9]$|^[a-zA-Z0-9][a-zA-Z0-9_-]{0,30}[a-zA-Z0-9]$')->min(1)->max(32);
```

Comment:
The regex itself says at least one character, but not more than 32, so what does `->min(1)->max(32)` add to it? Please motivate or remove.

Reply:
Removed